pkg/base/k8s/kubeclient.go

```diff
@@ -382,6 +383,30 @@ func (k *KubeClient) GetSystemDriveUUIDs() []string {
 	return drivesUUIDs
 }
 
+// GetNodeUUIDFromCSIBMNodeCR parse list of bmNodeCRs and find node UUID matching with nodeName
+func (k *KubeClient) GetNodeUUIDFromCSIBMNodeCR(nodeName string) (string, error) {
+	k8sNode := coreV1.Node{}
+	if err := k.Get(context.Background(), k8sCl.ObjectKey{Name: nodeName}, &k8sNode); err != nil {
+		return "", err
+	}
+
+	// get all existed bmNodes in cluster
+	bmNodeCRs := new(nodecrd.NodeList)
```

Comment:
Is it possible to avoid listing of all CRs?

Reply:
removed